csi/volumeutils.py

```diff
@@ -1135,6 +1135,19 @@ def mount_glusterfs_with_host(volname, mountpoint, hosts, options=None, is_clien
     if not os.path.exists(mountpoint):
         makedirs(mountpoint)
 
+    # FIXME: make this better later (an issue for external contribution)
+    # opt_array = None
+    # if options:
+    #     opt_array = []
+    #     for opt in options.split(","):
+    #         if not opt or opt == "":
+    #             break
+    #         for k,v in opt.split("="):
```

Comment:
Use `opt.split("=", 1)` to avoid spliting into more parts when value contains `=`. (For example: `xlator-option=*-posix.health-check-interval=0`)